### ISSUE TITLE
count=(>16) - where is this LIMIT fallback config?

### ISSUE THREAD

**Issue by VIEWSION** [apioo/fusio#162]
Hi,
just installed it yesterday and playing around a bit.... looks good so far! I quickly tested some action / route stuff with the built in SQL Action Class. Why are the results limited to 16 and how can I overwrite this? Adding the parameter count=10 works, but all over 16 not. Where is this configured?

Thx!

**Comment by chriskapp** [comment]
Hi, sounds great. So the "count" query parameter is limited to 32 and if the count is above 32 it defaults to 16. We do this because of security reasons but we could also easily adjust this limit to a higher max number if needed.

**Comment by VIEWSION** [comment]
OK - but even 32 is pretty low. Sure - it depends on the amount of the single record data, but for most cases I would set it to something like 250, 500 or so. Anyway - this should be configurable individually, no? It's more a feature request for simple and quick read only requests as it is possible to do the request by PHP / v8 anyway...

**Comment by chriskapp** [comment]
Yes its indeed pretty low. I think your idea is great to add a config field `limit` at the action where it is possible to specify the default limit. The action uses then this field as default limit (where you can provide any number). The count query parameter can then only be used to lower the result count and
if not provided the configured limit is used. I.e. if you have configured limit 100 at your action a user could request `/my/endpoint?count=50` but `/my/endpoint?count=150` would default to 100 since its above the limit.
